scripts/upgrade

```diff
@@ -136,66 +130,37 @@ ynh_add_fpm_config
 # UPGRADE BAIKAL
 #=================================================
 
+# We keep this to allow upgrade the config file in case it needs to be changed.
+
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	#=================================================
 	# UPGRADE BAIKAL CONFIGURATION
 	#=================================================
-	ynh_script_progression --message="Upgrading Baïkal configuration..." --weight=2
+	ynh_script_progression --message="Upgrading the configuration file..." --weight=2
 
 	if [ -z "$password_hash" ]; then
 		password=$(ynh_app_setting_get --app=$app --key=password)
 		password_hash=$(echo -n admin:BaikalDAV:$password | md5sum | cut -d ' ' -f 1)
 		ynh_app_setting_set --app=$app --key=password_hash --value=$password_hash
 	fi
 
-	bk_conf="${final_path}/config/baikal.yaml"
 	timezone=$(cat /etc/timezone)
 	path=${path_url%/}
-	deskey=$(ynh_app_setting_get --app=$app --key=encrypt_key)
-	ynh_app_setting_set --app=$app --key=encrypt_key --value="$deskey"
-
-	ynh_add_config --template="../conf/baikal.yaml" --destination="$bk_conf"
-
-	#=================================================
-	# UPGRADE BAIKAL
-	#=================================================
-	ynh_script_progression --message="Upgrading Baïkal..."
-
-	# Run Baikal upgrade
-	php"${phpversion}" "${final_path}/bin/upgrade.sh"
```

Comment:
There is no more need to execute the `/bin/upgrade.sh` script ?
If not, maybe files in `sources/extra_files/app` should be removed